### DISCUSSION TITLE
Download of fast5 files

### DISCUSSION THREAD

**Discussion by Annie-GW** [GoekeLab/xpore#117] [Q&A] [answered]
Dear xPore team,
I have been trying to download the nanopore DRS human data that you have published with xPore in Nature Biotechnology. I can find the fastq files on SRA or ENA but not the fast5. 
For example:
https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=ERR4706158

I am pretty sure I am missing something but just couldn't figure this out. Would you be able to shed some light on how I can access the related fast5 files?
Any tips that you could share would be greatly appreciated.

Kind regards,
Annie

**Comment by Annie-GW** [comment]
Just want to provide an update that I have found the link to download the fast5 files. However, I cannot find the fast5 files of rep2 for both HEK293T-Mettl3-KO and HEK293T-WT (see the link below). Would you be able to advise how I can access these two fast5 files?
https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=ERP124567

Thank you.
Kind regards,
Annie

**Comment by mantonglaw** [comment, marked as answer]
You can find the files you mentioned in 
https://www.ebi.ac.uk/ena/browser/view/PRJEB40872?show=reads

**Reply by ploy-np** [reply]
Thank you @mantonglaw for your answer :)
